solab_prius:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  

  # Velocity Controllers ---------------------------------------
  rear_left_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: rear_left_wheel_joint
    pid: {p: 100, i: 0.0, d: 0, i_clamp: 0.0}
  rear_right_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: rear_right_wheel_joint
    pid: {p: 100, i: 0.0, d: 0, i_clamp: 0.0}
  front_left_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: front_left_wheel_joint
    pid: {p: 100, i: 0.0, d: 0, i_clamp: 0.0}
  front_right_wheel_velocity_controller:
    type: effort_controllers/JointVelocityController
    joint: front_right_wheel_joint
    pid: {p: 100, i: 0.0, d: 0, i_clamp: 0.0}


  # Position Controllers ---------------------------------------
  front_left_steer_position_controller:
    joint: front_left_steer_joint
    type: effort_controllers/JointPositionController
    pid: {p: 100, i: 0.5, d: 25}
  front_right_steer_position_controller:
    joint: front_right_steer_joint
    type: effort_controllers/JointPositionController
    pid: {p: 100, i: 0.5, d: 25}

